server/admin.go

```diff
@@ -1017,13 +1017,23 @@ func (s *adminServer) ClusterFreeze(
 			keys.LocalMax,    // freeze first range (meta1)
 		}
 
-		for _, freezeFrom := range freezeFroms {
+		for i, freezeFrom := range freezeFroms {
+			if err := stream.Send(&serverpb.ClusterFreezeResponse{
+				Message: fmt.Sprintf("freezing meta ranges [stage %d]", i+1),
```

Comment:
Since this is user facing, I'd like to keep it as terse as possible (I added the stages so that we would know where it got stuck, though).